hamburger.js

```diff
@@ -0,0 +1,18 @@
+const hamburger = document.querySelector('.hamburger');
+const navnavMenu = document.querySelector('.navnav-menu');
+const contactLink = document.querySelector('.contact-link');
+const aboutLink = document.querySelector('.about-link');
+const portfolioLink = document.querySelector('.portfolio-link');
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navnavMenu.classList.toggle('active');
+});
```

Comment:
- [ ] As requested by the previous code reviewer please when scrolling in your portfoli make it disappear when the menu is shown you can make it with this property 🤩🔥 so not to the the menu open and the user can scroll so where is the benfit of the menu since the menu should make it easier to navigate 🔥🤩💯✨

>  overflow-y: hidden

![image](https://user-images.githubusercontent.com/58553711/223398050-ccff795f-2c6e-46a3-a377-4abfec9986be.png)